src/modules/lists/lists.service.ts

```diff
@@ -56,6 +56,13 @@ export class ListsService {
     if (params.public) {
       filters.public = { public: params.public };
     }
+    if (filters.listId) {
+      filters._id = { _id: params.listId };
+    }
+
+    if (filters.isFavorite) {
```

Comment:
We might want to set `filters.isFavorite` as `false`, instead check if is undefined otherwise the `false` value will get ignored

Reply:
@crysfel I am not sure I get you here is this what you are suggesting. Set the default`filters.isFavorite` as false and then check if `params.isFavourite` exists and reassign the filter. Something like this 👇 
```
const filters: any = {
      isFavorite: false,
 };
if (params.isFavorite) {
      filters.isFavorite = params.isFavorite;
}
```

Reply:
This is what I mean: When calling this method like this:

```
const list: List[] = await this.listsService.findByUserId({ isFavorite: false });
```

Then... in the current implementation, it will get ignored and it won't apply the filter, because the `if` is checking for a truthy value, instead we need to check for undefined.

```suggestion
if (params.isFavorite !== undefined) {
```

This way, even if we set `params.isFavorite` to `false`, the code under that if will get executed, and that's exactly what we want.

Reply:
sure it makes sense now just took longer to get 😄